salt/salt.spec

```diff
@@ -15,8 +15,12 @@
 # Please submit bugfixes or comments via http://bugs.opensuse.org/
 #
 %global debug_package %{nil}
-
-%if 0%{?suse_version} >= 1320
+%if 0%{?suse_version} > 1500
+%global build_py3   1
+%global build_py2   0
+%global default_py3 1
+%else
+%if 0%{?suse_version} >= 1500
```

Comment:
This was changed from 1320 to 1500; 1500 matches much closer the comment below (# SLE 15); The only difference in matching of distros would be openSUSE 13.2 - which was part of that if branch so far and now falls off. but that is so long out of support, there is no reason to care